src/pages/Home.js

```diff
@@ -0,0 +1,9 @@
+import Groups from '../components/Groups/Groups';
+
+const Home = () => (
+  <div>
+    <Groups />
+  </div>
+);
+
+export default Home;
```

Comment:
- [ ] The Home page should have a list of categories that could be filtered using given parameters, e.g. category name. You should use a Filter stateless component to filter data retrieved from the API. Right now there is no option on your home page to filter the data